lib/controller.rb

```diff
@@ -52,6 +52,7 @@ class SlowFood < Sinatra::Base
 
   get '/' do
     @dishes = Dish.all
+    @dishes_by_category = @dishes.group_by{|h| h[:category]}
```

Comment:
I think you could simplify this with:
```
@dishes_by_category = Dish.all.group_by{|h| h[:category]}
```
and remove line 54 altogether

Reply:
You will need to make a small update to our view where you mane use of `@dishes` but than might as well use `@dishes_by_category`